modules/setup/setup.js

```diff
@@ -193,6 +184,31 @@ class Setup {
         });                     
     }
 
+    startExistingContainer() {
+        const container = docker.getContainer(CONTAINER_NAME);
+        container.start(async (error, data) => {
+            if(error) {
+                Logger.verbose(`Error occured while starting the nodes: ${error}`);
+
+                try {
+                    let response = await RpcRequest.checkNodeStatus(LOCAL_NODE_URL);
+                    if(response.protocol.startsWith('PsCARTHAG')){
+                        Logger.info('Nodes are in running state....');
+                    } else {
+                        Logger.error (`Error occured in previously started nodes, restart nodes after running 'tezster stop-nodes'.`);
+                    }
+                } catch (error) {
+                    Helper.errorLogHandler(`Error occured while starting the nodes${error}`, `Error occured in previously started nodes, restart nodes after running 'tezster stop-nodes'.`);
+                }
+
+                Logger.warn(`If facing any issue, fetch logs on your system using 'tezster get-logs' or try restarting the nodes after running 'tezster stop-nodes'.`);
+
```

Comment:
If there's error, why do you need to perform 'checking node-status' flow?
It doesn;t make sense.

Reply:
In case, user ran start-nodes, and nodes didn't started successfully but container will be up. I am checking for if container is up and nodes didn't get started, we will show error accordingly. This is the possibility scenario, hence I added that.